packages/phone-number-privacy/signer/README.md

```diff
@@ -99,6 +106,17 @@ Then check on the service to make sure its running:
 
 `docker logs -f {CONTAINER_ID_HERE}`
 
+#### Key rotations
+
+After a key resharing, signers should rotate their key shares as follows:
+
+1. Store the new key share in the keystore according to the naming convention specified in the [Keystores](#keystores) section above.
+2. Notify the combiner operator that your signer is ready for the key rotation.
+3. The combiner operator will update the combiner to request the new key share version via a custom request header field once all signers are ready.
+4. The signers will fetch the new key shares from their keystores upon receiving these requests.
+5. When the combiner operator sees that all signers are signing with the new key share and confirms that the system is healthy, signers will be instructed to delete their old key shares.
+6. After deleting their old key shares, signers should increment `PHONE_NUMBER_PRIVACY_LATEST_KEY_VERSION` or `DOMAINS_LATEST_KEY_VERSION` accordingly. This will instruct the signer to prefetch this key version the next time it starts up.
```

Comment:
Good point. It doesn't make a big difference when the signer updates these variables since prefetching is really just an optimization. I'll make it step 2